docs/basics.adoc

```diff
@@ -344,10 +344,9 @@ This section is a brief overview of the Gremlin query language. For more informa
 * http://tinkerpop.apache.org/docs/{tinkerpop_version}/reference[Complete Gremlin Manual]: Reference manual for all of the Gremlin steps.
 * http://tinkerpop.apache.org/docs/{tinkerpop_version}/tutorials/the-gremlin-console/[Gremlin Console Tutorial]: Learn how to use the Gremlin Console effectively to traverse and analyze a graph interactively.
 * http://tinkerpop.apache.org/docs/{tinkerpop_version}/recipes/[Gremlin Recipes]: A collection of best practices and common traversal patterns for Gremlin.
-* http://tinkerpop.apache.org/index.html#language-drivers[Gremlin Language Drivers]: Connect to a Gremlin Server with different programming languages, including Go, JavaScript, .NET/C#, PHP, Python, Ruby, Scala, and TypeScript.
```

Comment:
I think we could just leave the link to TinkerPop references for supported drivers.

The additional documentation you added in this PR on how specifically Gremlin can be used to query JanusGraph should help funnel the information from generic TInkerPop to JanusGraph.